docs/operation/cluster_and_service_configuration.rst

```diff
@@ -30,12 +30,19 @@ MetalK8s installation.
 
 To add a new static user, perform the following operations:
 
-#. Generate a password ``hash``. The command below requires **apache-utils**
-   package to be installed.
+#. Generate a bcrypt hash of your new password.
+
+   - Startup the MetalK8s utility container by refering to
+     :ref:`this procedure <Troubleshooting-container>`
+
+   - To generate the bcrypt hash, run the following command while replacing
+     the <password> field with your plaintext password.
 
    .. code-block:: shell
 
-      root@bootstrap $ htpasswd -bnBC 12 "" <replace-with-password> | tr -d ':\n'
+      root@bootstrap $ crictl exec -it \
```

Comment:
Nah. The procedure to start the `metalk8s-utils` container (which I'll comment on separately) doesn't fix the Pod/container to a particular node, so `crictl` may not even work.

Also, it's not great form to pass passwords as CLI arguments, they easily end up in shell history and whatnot.

So, instead, why not

```
kubectl run htpasswd-$$ -i --image THE_IMAGE_WE_WANT --restart=Never --rm --command -- sh -c "htpasswd -nBC 12 '' | tr -d ':'"
```

Reply:
Thinking of it, that won't work with taints.

Now, why don't we install `httpd-utils` on the bootstrap node, and document this package is required (in case someone wants to run this on another node)?

Reply:
Ok, i will add the package to the bootstrap node and be done

Reply:
Done